src/app/pages/ArticlePage/ArticlePage.tsx

```diff
@@ -160,59 +234,30 @@ const ArticlePage = ({ pageData }: { pageData: Article }) => {
     audio: MediaLoader,
     video: MediaLoader,
     text,
-    image: (props: ComponentToRenderProps) => (
-      <ImageWithCaption
-        {...props}
-        sizes="(min-width: 1008px) 760px, 100vw"
-        shouldPreload={preloadLeadImageToggle}
-      />
+    image: getImageComponent(preloadLeadImageToggle),
+    timestamp: getTimestampComponent(
+      hasByline,
+      bylineContribBlocks,
+      firstPublished,
+      lastPublished,
     ),
-    timestamp: (props: ComponentToRenderProps & TimeStampProps) =>
-      hasByline ? (
-        <Byline blocks={bylineContribBlocks}>
-          <Timestamp
-            firstPublished={new Date(firstPublished).getTime()}
-            lastPublished={new Date(lastPublished).getTime()}
-            popOut={false}
-          />
-        </Byline>
-      ) : (
-        <Timestamp {...props} popOut={false} />
-      ),
     social: SocialEmbedContainer,
```

Comment:
Curious if this will help solve re-rendering issues that are (maybe?) hidden from us that this lint is exposing.

We have these pages memoised so that when the user uses the `Skip to content` button, the likes of the Social Embeds don't re-render. However we already have the `SocialEmbedContainer` component in the "correct" format according to the linter, but it still re-renders if we remove the memoisation logic.

Reply:
I'll have a look into this to see if it makes a difference

Reply:
Had a mess about with the React profiler, it still re-renders in ArticlePage, you can see the OptmizelyPageViewTracking being re-renderer here, and all I'm doing is scrolling down the page after it's fully loaded. It's probably fine as performance isn't really noticable, DOM updates aren't really happening at all, but this would be great to fix so we can prevent re-render issues like the one you mentioned, without having to pepper useMemo everywhere. Gonna have a look today
![image](https://github.com/user-attachments/assets/cd619378-d787-478f-b059-34d5fb3b4163)

Reply:
Ahh wait nevermind, that's because of the scroll tracking